app/locales/en.json

```
@@ -119,7 +119,7 @@
     "launch_notif_subheader": "We won't bother you except to share updates on your potential exposure risks.",
     "launch_notification_access": "Allow notifications",
     "launch_screen1_header": "The way back to normal starts here.",
-    "launch_screen2_header": "Get notified if you cross paths with someone later diagnosed for COVID-19.",
+    "launch_screen2_header": "Get notified if you have been in contact with someone who may be COVID-19 positive after having contact.",
```

The last part no made sence, please delete it
```suggestion
    "launch_screen2_header": "Get notified if you have been in contact with someone who may be COVID-19 positive.",
```